src/graph/expression_graph.h

```diff
@@ -739,10 +756,13 @@ class ExpressionGraph : public std::enable_shared_from_this<ExpressionGraph> {
 
 public:
   /** Load model (mainly parameter objects) from array of io::Items */
-  void load(std::vector<io::Item>& ioItems, bool markReloaded = true) {
+  void load(const std::vector<io::Item>& ioItems, bool markReloaded = true, bool dropF0prefix = false) {
```

Comment:
Why the new argument it needed? If it really needs to be added to this function, can you please add a one-line comment into the code below and maybe update the doxygen doc as well?

Reply:
Those `"F0::"` prefixes get added when preparing the `io::Item`s so that they can be swapped into the translation graph. I'd have preferred to not let this logic leak out of `swappable.cpp` but unfortunately it seems that it would require modifying the `io:Item` names in place each time before `ExpressionGraph::load` or `GPUEngineTranslate::SwapPointers` is called. I think it justifies doing this upon loading the `io::Item`s instead.